src/components/AdContainer/ad-container.scss

```diff
@@ -0,0 +1,29 @@
+@import 'globals';
+
+.root {
+  max-width: 100%;
+  position: relative;
+  font-size: 12px;
+  font-weight: 700;
+}
+
+.root * {
+  max-width: 100%;
+}
+
+.wideboard {
+  display: table;
+  margin: 2.5em auto;
+
+  &:empty {
+    display: none;
+  }
+}
+
+.rectangle {
+  margin-bottom: 1.25em;
+
+  &:empty {
+    display: none;
+  }
+}
```

Comment:
As discussed on Friday I would not include any default styling for ad slot. This should be done by the actual implementation in the theme.

@mattberridge: What is your opinion on this?